server/accessmanager.go

```diff
@@ -0,0 +1,55 @@
+package server
+import (
+	"github.com/smancke/guble/guble"
+	"fmt"
+	"net/http"
+	"io/ioutil"
+)
+
+
+type AllowAllAccessManager bool
+
+func NewAllowAllAccessManager(allowAll bool) AllowAllAccessManager {
+	return AllowAllAccessManager(allowAll)
+}
+
+func (am AllowAllAccessManager) AccessAllowed(accessType AccessType, userId string, path guble.Path) bool {
+	return bool(am);
+}
+
+type RestAccessManager string
+
+func NewRestAccessManager(url string) RestAccessManager {
+	return RestAccessManager(url)
+}
+
+func (am RestAccessManager) AccessAllowed(accessType AccessType, userId string, path guble.Path) bool {
+	url := string(am) + "?type="
```

Comment:
I would use the URL type to add parameters.
This is more robust and allows the used url to already have parameters inside:
https://golang.org/pkg/net/url/#example_URL